### PR TITLE
KFS CSV

### PR DESCRIPTION
**Pull request by robstolarz**
Adds a new button...
screen shot 2018-04-27 at 12 06 02 pm
That generates a Kuali compatible CSV.screen shot 2018-04-27 at 12 06 27 pm

### CODE REVIEW

File: vendor/engines/nucore_kfs/app/controllers/kfs_export_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class KfsExportController < ApplicationController
+  def show
+    @journal = Journal.find(params[:id])
+    @journal_rows = @journal.journal_rows
+    headers["Content-Disposition"] = "attachment;"
```
Comment (jhanggi) [inline vendor/engines/nucore_kfs/app/controllers/kfs_export_controller.rb]: We use `CSVHelper#set_csv_headers` in other places for setting the content disposition (and it allows you to specify a custom filename as well for the download).

**Comment by gluxon** [comment]
@jhanggi Just wanted to say thanks for the review. We weren't quite expecting one from you or Table XI, but it's really appreciated regardless.

**Comment by robstolarz** [comment]
Killed off `vendor/engines/nucore_kfs/test/dummy/`, added `CSVHelper#set_csv_headers`

**Comment by joshea0** [comment]
It looks like this PR has gone stale. Since I've been asked to take over the KFS integration I've opened a new PR #76 to get a fresh start.

**Pull request closed without merging**